pipeline/terraform/scripts/create_pipeline_storage_users.py

```diff
@@ -79,7 +79,7 @@ def create_roles(es, index):
         role_name = f"{index}_{role_suffix}"
 
         es.security.put_role(
-            role_name,
+            name=role_name,
```

Comment:
This is a change for the newest version of the Elasticsearch Python libraries; it insists on keyword arguments.